project.clj

```diff
@@ -0,0 +1,19 @@
+(defproject puppetlabs/jruby-utils "0.1.0-SNAPSHOT"
+  :description "A library for working with JRuby"
+  :url "https://github.com/puppetlabs/jruby-utils"
+  :license {:name "Apache License, Version 2.0"
+            :url "http://www.apache.org/licenses/LICENSE-2.0"}
+
+  :pedantic? :abort
+
+  :dependencies [[org.clojure/clojure "1.7.0"]]
+
+  :lein-release {:scm         :git
+                 :deploy-via  :lein-deploy}
+
+  :deploy-repositories [["releases" {:url "https://clojars.org/repo"
+                                     :username :env/clojars_jenkins_username
+                                     :password :env/clojars_jenkins_password
+                                     :sign-releases false}]]
+
+  :plugins [[lein-release "1.0.5"]])
```

Comment:
I don't think this is necessary, lein now has the release task built-in.  We probably haven't cleaned this up from other projects where we added it before that version of lein.